tools/testsys/src/restart_test.rs

```diff
@@ -0,0 +1,20 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use model::test_manager::TestManager;
+
+/// Restart an object from a testsys cluster.
```

Comment:
```suggestion
/// Restart a test. This will delete the test object from the testsys cluster and replace it with
/// a new, identical test object with a clean state.
```